.pylintrc

```diff
@@ -15,8 +15,48 @@ msg-template='{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}'
 # C0411: Import order
 # C0111: Missing docstring
 # W0613: Unused argument
-# E1101: Seems to have a bug. Says numpy.random has no member RandomState
-# E0611: Seems to have a bug. Complains about imports of tensorflow
+# E1101: pylint seems to have a bug. Says numpy.random has no member RandomState
+# E0611: pylint seems to have a bug. Complains about imports of tensorflow
 # R0913: Too many arguments
 # R0914: Too many local variables
-disable=C0103,C0411,C0111,W0613,E1101,E0611,R0913,R0914
+# W0611: Unused import
```

Comment:
Why disable this one?

Reply:
Just to keep this PR smaller. We should get it back on soon: https://github.com/tensorflow/cleverhans/issues/598

Reply:
SGTM